pyanaconda/ui/gui/spokes/password.py

```diff
@@ -39,7 +39,12 @@ class PasswordSpoke(FirstbootSpokeMixIn, NormalSpoke, GUISpokePasswordCheckHandl
     builderObjects = ["passwordWindow"]
 
     mainWidgetName = "passwordWindow"
-    focusWidgetName = "pw"
+    # Don't focus on the password entry field by default as is some cases
+    # a password might have been set in kickstart and focusing on the password
+    # filed might clear it (& hide the "password is set" message).
+    # Also there is logic in place that should focus the password entry field if
+    # password is *not* set in kickstart.
+    focusWidgetName = "passwordWindow"
```

Comment:
I don't think that's needed - all the other spokes do the same thing & we are on the rhel7-branch. But it might make sense to refactor these out to local constants on the master branch.